NineChronicles.Headless/GraphTypes/Diff/StateDiffType.cs

```diff
@@ -0,0 +1,51 @@
+using Bencodex;
+using Bencodex.Types;
+using GraphQL.Types;
+using Libplanet.Common;
+
+namespace NineChronicles.Headless.GraphTypes.States;
+
+public class StateDiffType : ObjectGraphType<StateDiffType.Value>
+{
+    public class Value
+    {
+        public string Path { get; }
+        public IValue BaseState { get; }
+        public IValue? ChangedState { get; }
+
+        public Value(string path, IValue baseState, IValue? changedState)
+        {
+            Path = path;
+            BaseState = baseState;
+            ChangedState = changedState;
+        }
+
+        public bool IsAddressRelated(string address) => Path.Contains(address);
```

Comment:
It seems unnecessary.

```suggestion
        }
```

Reply:
yes right 👍 thanks